neuraxle/steps/flow.py

```diff
@@ -77,6 +77,79 @@ def fit_transform(self, data_inputs, expected_outputs=None) -> 'ForceMustHandleM
 OPTIONAL_ENABLED_HYPERPARAM = 'enabled'
 
 
+class TrainOnlyWrapper(ForceMustHandleMixin, MetaStepMixin, BaseStep):
+    """
+    A wrapper to run wrapped step only in test mode
+
+    Execute only in test mode:
+
+    .. code-block:: python
+
+        p = Pipeline([
+            TrainOnlyWrapper(Identity())
+        ])
+
+    Execute only in train mode:
+
+    .. code-block:: python
+
+        p = Pipeline([
+            TrainOnlyWrapper(Identity(), test_only=False)
+        ])
```

Comment:
I would like 3 classes like that: 

- class TrainOrTestOnlyWrapper(is_train_only=True)
- class TrainOnlyWrapper(): `def __init__(self): TrainOrTestOnlyWrapper.__init__(self, is_train_only=True)`
- class TestOnlyWrapper(): `def __init__(self): TrainOrTestOnlyWrapper.__init__(self, is_train_only=False)`